Jenkinsfile

```diff
@@ -139,21 +139,29 @@ def checkout_git(label) {
         git checkout -qf FETCH_HEAD
         git rev-parse HEAD
         """
+        if (label.contains("windows")) {
+            bat script
+        } else {
+            sh """#!/bin/sh
+                set -xe
+                ${script}
+            """
+        }
     } else {
-        script = """
-        git clone --depth=1 https://github.com/pyca/cryptography.git cryptography
-        cd cryptography
-        git checkout ${env.BRANCH_NAME}
-        git rev-parse HEAD
-        """
-    }
-    if (label.contains("windows")) {
-        bat script
-    } else {
-        sh """#!/bin/sh
-        set -xe
-        ${script}
-        """
+        checkout([
+            $class: 'GitSCM',
+            branches: [[name: '*/master']],
```

Comment:
I don't think so. We might be able to do the interpolation trick we were trying previously?